dashboard/static/css/layout.css

```diff
@@ -22,9 +22,13 @@ body {
     color: #fff;
 }
 
-.center {
```

Comment:
This makes the new datadog status iframe un-centered. :cry: 

<img width="1440" alt="2016-03-02_1550" src="https://cloud.githubusercontent.com/assets/71928/13476679/9647f2ec-e08e-11e5-9768-d8de2620680a.png">